src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
```

Comment:
Given that this extends the base button rather than a fab, it will likely need styling for its disabled state. Is this covered somewhere?